update

```diff
@@ -124,19 +133,21 @@ function makeEntry (dir, parsedFileName, oldList) {
     }
   }
 
-  if (!build.keccak256 || !build.urls) {
+  if (!build.keccak256 || !build.urls || build.urls.length != 2) {
     const fileContent = fs.readFileSync(absolutePath)
     build.keccak256 = '0x' + ethUtil.keccak(fileContent).toString('hex')
-    build.urls = ['bzzr://' + swarmhash(fileContent).toString('hex')]
-
-    console.log("Hashing '" + pathRelativeToRoot + "'")
+    console.log("Computing hashes of '" + pathRelativeToRoot + "'")
+    build.urls = [
+      'bzzr://' + swarmhash(fileContent).toString('hex'),
+      'dweb:ipfs/' + await ipfsHash(fileContent)
```

Comment:
I'm not all that familar with IPFS and I had too look it up so I might be completely wrong but isn't the scheme `dweb:/ipfs/$hash`?

https://github.com/ipfs/specs/tree/addressing/dweb-addressing#interoperability

Reply:
Thanks a lot! Updated.